boards/arm/efm32pg_stk3402a/doc/index.rst

```diff
@@ -189,6 +189,14 @@ terminal session:
 
    Hello World! arm
 
+Twister Testing on EFM32PG-STK3402A
+===================================
+
+The Twister tests can be run as follows:
+
+.. code-block:: console
+
+   scripts/twister --device-testing --hardware-map <relative path to>-map.yml
```

Comment:
IMO this is not needed in the board doc.

Reply:
That depends on what we have to do to solve the problems that we have.